docs/Sonarr/sonarr-collection-of-custom-formats.md

```diff
@@ -141,9 +141,11 @@ I also made 3 guides related to this one.
 
 ### DTS X
 
-??? question "DTS X - [Click to show/hide]"
+??? question "DTS:X - [Click to show/hide]"
 
-    DTS:X is an object-based audio codec, which aims to create a multi-dimensional sound that “moves around you like it would in real life”
+    [DTS:X is an object-based audio codec](https://www.whathifi.com/advice/dtsx-what-it-how-can-you-get-it), which aims to create a multi-dimensional sound that “moves around you like it would in real life”
```

Comment:
```suggestion
    [DTS:X is an object-based audio codec](https://www.whathifi.com/advice/dtsx-what-it-how-can-you-get-it), which aims to create a multi-dimensional sound that “moves around you like it would in real life”.
```